core/templates/services/context.service.spec.ts

```diff
@@ -244,14 +319,59 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
```

Comment:
Duplicate description. Please make it more specific.

Reply:
Done!